app/renderer/js/utils/dnd-util.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const ConfigUtil = require(__dirname + '/config-util.js');
+
+function dnd() {
+	this.initDNDButton();
+	const dnd = !ConfigUtil.getConfigItem('dnd', false);
+	const settingList = ['showNotification', 'silent', 'flashTaskbarOnMessage'];
+	let oldSettings;
+	if (dnd) {
+		const currentSettings = {};
+		oldSettings = {};
```

Comment:
I'd do change the `currentSetting` -> `oldSetting` and `oldSettings` to `newSettings` or `dndSetting`.  They are used in way seems confusing.